### DISCUSSION TITLE
Block command for X milliseconds

### DISCUSSION THREAD

**Discussion by Symbai** [dotnet/wpf#7849] [Q&A]
How can I block a command in MVVM (no third party implementation!) for X milliseconds to avoid its being accidentally executed twice by double press / click?

At the moment I have to create property for the CanExecute command function which I change with delay but I wonder if this can be done any better with less code.

**Comment by miloush** [comment]
Remember the last time it was called and do nothing if it hasn't been long enough? I wouldn't use CanExecute because it is reflected in UI, key bindings etc.
